ethers-signers/Cargo.toml

```diff
@@ -28,6 +28,7 @@ yubihsm = { version = "0.39.0", features = ["secp256k1", "http", "usb"], optiona
 futures-util = "0.3.18"
 futures-executor = "0.3.18"
 semver = "1.0.4"
+trezor-client = { package="trezor", git = "https://github.com/joshiedo/rust-trezor-api", rev = "f0c9775", optional = true }
```

Comment:
this would prevent us from publishing a new version of ethers, 
what's the state of that crate, any chance you can publish this beforehand?

Reply:
done: [trezor-client](https://crates.io/crates/trezor-client)